qemu/tests/cfg/block_kill_reconnect_with_remote_storage.cfg

```diff
@@ -20,7 +20,7 @@
     get_disk_cmd = "ls /dev/[hsv]d[a-z]* | sort"
     disk_op_cmd = "dd if=/dev/zero of=%s bs=1k count=1000 oflag=direct &&"
     disk_op_cmd += " dd if=%s of=/dev/null bs=1k count=1000 iflag=direct"
-    find_system_disk_cmd = "df -h | grep /boot | awk -F "[0-9] " '{print $1}'"
+    find_system_disk_cmd = "df -h | grep '/boot$' | awk -F "[0-9] " '{print $1}'"
```

Comment:
This may cause error when /boot disk is a nvme disk, e.g. 
```
$ df -h | grep '/boot$' | awk -F "[0-9] " '{print $1}'
/dev/nvme0n1p
```
It should be /dev/nvme0n1 in the example above.

Reply:
or:
lsblk -dnpo PKNAME `df -h | grep '/boot$' | awk '{print $1}'`

Not sure if the cartesian parser can work with this.

Reply:
Hi Zhenchao,

Thanks for your very detailed info. The `df -h | grep '/boot$' | awk '{print $1}' | xargs lsblk -dnpo PKNAME` works for me.

But I am just thinking whether it's necessary to consider NVMe disk here, as the command here is for VMs, and the only scenario for NVME disk is paththrough, but I have no ideas what the scenarios are for paththroughing a NVMe disk as system disk in a VM.

What your points here?

Reply:
And the intentional of this command is to find the data disk in the line below `find_data_disk_cmd = "ls /dev/[vhs]d* | grep -v %s"`, which is just focus on `[vhs]` disks.

Reply:
Hi @hellohellenmao , I checked this on host OS which employs a nvme disk for boot, it's safe for us to run the command inside a VM, I am OK with your change.